core/tx_pool.go

```diff
@@ -601,42 +605,46 @@ func (pool *TxPool) local() map[common.Address]types.Transactions {
 func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 	// Heuristic limit, reject transactions over 32KB to prevent DOS attacks
 	if tx.Size() > 32*1024 {
```

Comment:
Does this number still make sense?  Maybe open a follow up ticket for further investigation.

Reply:
Opened a ticket, #2154